src/migrations/migrationHelpers.ts

```diff
@@ -12,7 +13,7 @@ export const collectVuexState = (() => {
     if (window?.browser) {
       if (isCollecting.value) {
         await watchUntilTruthy(isCollecting);
-      } else if (!vuexState) {
+      } else if (!vuexState && !IS_OFFSCREEN_TAB) {
```

Comment:
Are you sure the offscreen should not share the state with the extension?

Reply:
Offscreen uses `localStorage` instead of `window.browser.storage.local`
and IIRC syncing happens in `WalletStorage.ts`